src/sagas/index.js

```diff
@@ -461,28 +453,56 @@ export function* fetchVmSnapshots ({ vmId }) {
   let snapshotsInternal = []
 
   if (snapshots && snapshots.snapshot) {
-    snapshotsInternal = snapshots.snapshot.map(snapshot => Transforms.Snapshot.toInternal({ snapshot }))
-
-    // NOTE: Snapshot Disks and Nics are not currently (Sept-2018) available via
-    //       additional/follow param on the snapshot fetch.  We need to fetch them
-    //       directly.
-    for (const snapshot of snapshotsInternal) {
-      const follows = yield all([
-        call(fetchVmSnapshotDisks, { vmId, snapshotId: snapshot.id }),
-        call(fetchVmSnapshotNics, { vmId, snapshotId: snapshot.id }),
-      ])
-      snapshot.disks = follows[0]
-      snapshot.nics = follows[1]
-    }
+    snapshotsInternal = snapshots.snapshot.map(snapshot => Api.snapshotToInternal({ snapshot }))
```

Comment:
@sjd78 
Whenever trying to delete,create a snapshot, the following exception occurs:
![image](https://user-images.githubusercontent.com/18169498/125444732-61e59f6a-7cd4-421f-918f-222ba9d26fd4.png)

It seems that Api.snapshotToInternal => Transforms.Snapshot.toInternal.
Please fix on a follow up PR.